.github/ISSUE_TEMPLATE/bug.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug Report
+about: Report a bug encountered while operating MetalK8s
+labels: bug,moonshot
```

Comment:
We'll need to find something better than `moonshot` for labels to manage different versions (if people want to report bugs on MetalK8s 1, they/we could be surprised), maybe even asking for affected version(s) in the issue body. Probably not necessary for this PR though, but it needs to be tracked at least.

Reply:
Tracked by #1097